javaee/impl/src/main/java/org/jboss/forge/addon/javaee/security/ui/AddRoleCommand.java

```diff
@@ -0,0 +1,65 @@
+package org.jboss.forge.addon.javaee.security.ui;
+
+import org.jboss.forge.addon.facets.constraints.FacetConstraint;
+import org.jboss.forge.addon.javaee.servlet.ServletFacet;
+import org.jboss.forge.addon.javaee.ui.AbstractJavaEECommand;
+import org.jboss.forge.addon.ui.command.PrerequisiteCommandsProvider;
+import org.jboss.forge.addon.ui.context.UIBuilder;
+import org.jboss.forge.addon.ui.context.UIContext;
+import org.jboss.forge.addon.ui.context.UIExecutionContext;
+import org.jboss.forge.addon.ui.input.UIInput;
+import org.jboss.forge.addon.ui.metadata.UICommandMetadata;
+import org.jboss.forge.addon.ui.metadata.WithAttributes;
+import org.jboss.forge.addon.ui.result.NavigationResult;
+import org.jboss.forge.addon.ui.result.Result;
+import org.jboss.forge.addon.ui.result.Results;
+import org.jboss.forge.addon.ui.result.navigation.NavigationResultBuilder;
+import org.jboss.forge.addon.ui.util.Categories;
+import org.jboss.forge.addon.ui.util.Metadata;
+
+import javax.inject.Inject;
+
+/**
+ * Adds a security role to the current project.
+ *
+ * @author <a href="mailto:ivan.st.ivanov@gmail.com">Ivan St. Ivanov</a>
+ */
+@FacetConstraint(ServletFacet.class)
+public class AddRoleCommand extends AbstractJavaEECommand
```

Comment:
I can add the two commands (remove role and remove constraint) with the next pull request. I have a question here though. I intentionally stopped at the roles, login configs and constraints with this pull request and left the other features for subsequent commits (with their own sub-tasks). So if you think we need more functionalities, we may consider creating a different sub-task?